### PR TITLE
Landingpage

### LINKED ISSUE #3: [FEATURE] Landing Page
**Add landing page**
This is to show the service related data such as the intro and about content  plus some testmonials

### PR DESCRIPTION
**Pull request by briantical**
## Description
Created the landing page for the application
Installed the react for the UI.
List any dependencies that are required for this change.

Fixes #3 

## How Has This Been Tested?
No tests performed so far

## Screenshots (if applicable, else remove this line / section)
Screenshot 2020-05-23 at 13 20 57

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/views/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
+.content {
+  width: 100%;
+  position: absolute;
+  top: 0;
+  bottom: 96px;
+}
+
```
Comment (michaelgobz) [inline src/views/Home/Home.css:7]: before we merge we need to make it responsive so its pending

Comment (briantical) [reply src/views/Home/Home.css:7]: Kindly provide the designs for the mobile view for me to proceed

Comment (michaelgobz) [reply src/views/Home/Home.css:7]: please check the slack channel 

https://www.figma.com/file/s7kKafXNQ6bu8LF8IIIi3H/UGRO-MOBILE-00?node-id=0%3A1

Comment (briantical) [reply src/views/Home/Home.css:7]: Noted
I will update you shortly

**Pull request closed without merging**